src/host/ConsoleArguments.cpp

```diff
@@ -21,6 +21,79 @@ const std::wstring_view ConsoleArguments::INHERIT_CURSOR_ARG = L"--inheritcursor
 const std::wstring_view ConsoleArguments::FEATURE_ARG = L"--feature";
 const std::wstring_view ConsoleArguments::FEATURE_PTY_ARG = L"pty";
 
+std::wstring escape_argument(std::wstring_view ac)

```

Comment:
```suggestion
std::wstring EscapeArgument(std::wstring_view ac)
```

It doesn't have to be this naming exactly, but I don't think we have any functions in our codebase that we defined with all lowercase names with an underscore pattern.